common/request_handler.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+import requests
+from requests.exceptions import ConnectionError
+from structlog import wrap_logger
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def request_handler(method, url, auth=None, headers=None, json=None, data=None, files=None, fail=True):
```

Comment:
Seems overkill to add this in when we only need to `requests.post(url, auth=auth, files=files)`

Reply:
files, fail and json are the only ones not being used atm. I've removed fail but kept files and json since there will be use cases for them in the future.

Reply:
Oh i think i misunderstood, i agree probably no need for a request handler... i'll remove it.

Reply:
The only reason we've had request handlers in the other repos is so we can wrap `RequestException` errors into one function. Otherwise we need a try/except every time we make a request somewhere. Not sure what the best way to go here is though